docs/Project-Configuration/Project-Commands.md

```diff
@@ -2,6 +2,30 @@
 
 Project Commands are the fundamental units of functionality in an Evergreen task.
 
+## Basic Command Structure
+
+```yaml
+- command: shell.exec
+  display_name: run my cool script ## optional 
+  type: system ## optional
+  timeout_secs: 10 ## optional
+  retry_on_failure: true ## optional
+  params:
+    script: echo "my script"
+```
+Explanation:
+
+-   `command`: a command name from the predefined set of commands documented below.
+-   `display_name`: an optional user defined display name for the command. This will show up in logs and in the UI
+     with more details, for example: ~'shell.exec' ('run my cool script') (step 1 of 1)~
```

Comment:
Sorry, I meant backticks, not tildes... 🤦  Tildes in Markdown produce a strikethrough, backticks produce a string literal.